### PR TITLE
ON-1148 mendoza import

### PR DESCRIPTION
**Pull request by mfonsecaOEF**
This PR includes:
- extraction script 
- transformation script
for stationary energy for Mendoza cities

### CODE REVIEW

File: global-api/migrations/versions/88e4463fef06_locode_data_table.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+# revision identifiers, used by Alembic.
+revision: str = '88e4463fef06'
+down_revision: Union[str, None] = 'cc8d00d4372e'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    op.create_table(
+        "city_locode",
```
Comment (evanp) [inline global-api/migrations/versions/88e4463fef06_locode_data_table.py:24]: Not a great table name! How about "citywide_emissions"?

File: global-api/routes/city_locode_endpoint_mendoza_stationary_energy.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
+from fastapi import APIRouter, HTTPException
+from sqlalchemy import text
+import pandas as pd
```
Comment (evanp) [inline global-api/routes/city_locode_endpoint_mendoza_stationary_energy.py:3]: Booo! No Pandas in API endpoints!

File: global-api/routes/city_locode_endpoint_mendoza_stationary_energy.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+@api_router.get("/source/{source_name}/city/{locode}/{year}/{GPC_refno}")
+def get_emissions_by_locode_and_year(source_name: str, locode: str, year: str, GPC_refno: str):
+
+    records = db_query(source_name, locode, year, GPC_refno)
+
+    if not records:
+        raise HTTPException(status_code=404, detail="No data available")
+
+    df = pd.DataFrame(records)
```
Comment (evanp) [inline global-api/routes/city_locode_endpoint_mendoza_stationary_energy.py:36]: Come now! You get an array, convert it to a data frame, and then convert it back to an array with iterrows?

I think we can do this a little simpler.

**Review by evanp — CHANGES_REQUESTED** [review]
I gave some feedback. I think these need to be fixed before we merge.

File: global-api/migrations/versions/88e4463fef06_locode_data_table.py (hunk 1 above)
Comment (mfonsecaOEF) [reply global-api/migrations/versions/88e4463fef06_locode_data_table.py:24]: it sounds better. I'm going to change it.

File: global-api/routes/city_locode_endpoint_mendoza_stationary_energy.py (hunk 2 above)
Comment (mfonsecaOEF) [reply global-api/routes/city_locode_endpoint_mendoza_stationary_energy.py:3]: changed!

**Pull request merged**